libraries/uchiwa_helpers.rb

```diff
@@ -1,9 +1,25 @@
 module Uchiwa
   module Helpers
 
+    extend ChefVaultCookbook if Kernel.const_defined?("ChefVaultCookbook")
     def self.data_bag_item(data_bag_name, data_bag_item, missing_ok=false)
 
-      Chef::EncryptedDataBagItem.load(data_bag_name, data_bag_item).to_hash.delete_if { |key, value| key == 'id' }
+      raw_hash = Chef::DataBagItem.load(data_bag_name, data_bag_item)
+      encrypted = raw_hash.detect do |key, value|
```

Comment:
This seems like a much simpler way of doing this and comes from the readme so I think we should follow the same pattern: https://github.com/chef/chef-vault#determining-the-type-of-a-data-bag-item.

```
case ChefVault::Item.data_bag_item_type('data_bag_name', 'data_bag_item')
when :normal
  ...
when :encrypted
  ...
when :vault
  ...
end
```

Reply:
@majormoses Thanks for the suggestion. I have pushed changes.